share/transformers/org_datacite.py

```diff
@@ -490,7 +494,7 @@ def get_schema(self, type):
     )
     description = tools.RunPython(
         force_text,
-        tools.Try(ctx.record.metadata['oai_datacite'].payload.resource.descriptions.description[0])
+        tools.Try(ctx.record.metadata['oai_datacite'].payload.resource.descriptions)
```

Comment:
You might want to use `tools.Join` here. If there are multiple descriptions it will join them all into a single string.